cms/templates/widgets/_ui-dnd-indicator-after.html

```diff
@@ -1,2 +1,2 @@
 <%page expression_filter="h"/>
-<span class="draggable-drop-indicator draggable-drop-indicator-after"><i class="icon fa fa-caret-right"></i></span>
+<span class="draggable-drop-indicator draggable-drop-indicator-after"><span class="icon fa fa-caret-right" aria-hidden="true"></span></span>
```

Comment:
These files weren't related to the DnD component-- they were for reordering sections, subsections, and units on the Studio course outline page. But these templates have now moved directly into outline.underscore and xblock-outline.underscore. We need your help with those templates, @clrux.